concent_api/core/views.py

```diff
@@ -374,6 +403,13 @@ def validate_golem_message_reject(data):
             raise Http400("Wrong type of deadline field.")
 
 
+def validate_golem_message_timestamp(timestamp):
+    try:
+        verify_time(timestamp)
+    except (MessageFromFutureError, MessageTooOldError, TimestampError):
+        raise Http400('Message timestamp is wrong.')
```

Comment:
Please return the original exception message instead of replacing it with a generic one that does not even say what's wrong with the timestamp.

Reply:
:+1: